Description: Customize the /etc/dma configuration files for Debian.
 On Debian, the aliases file lives in /etc, not /etc/mail.
 Define a smarthost by default, albeit an invalid one.
 Use deferred delivery - the package includes a cron job running the queue.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: not-needed
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-06-02

--- a/dma.8
+++ b/dma.8
@@ -166,7 +166,7 @@
 .Pa dma.conf .
 .Bl -tag -width 4n
 .It Ic SMARTHOST Xo
-(string, default=empty)
+(string, default=mail.example.com)
 .Xc
 If you want to send outgoing mails via a smarthost, set this variable to
 your smarthosts address.
@@ -181,7 +181,7 @@
 will deliver all mails to this port, regardless of whether a smarthost is set
 or not.
 .It Ic ALIASES Xo
-(string, default=/etc/mail/aliases)
+(string, default=/etc/aliases)
 .Xc
 Path to the local aliases file.
 Just stick with the default.
--- a/etc/dma.conf
+++ b/etc/dma.conf
@@ -2,13 +2,15 @@
 #
 # Your smarthost (also called relayhost).  Leave blank if you don't want
 # smarthost support.
-#SMARTHOST
+# NOTE: on Debian systems this is handled via debconf!
+# Please use dpkg-reconfigure dma to change this value.
+SMARTHOST mail.example.com
 
 # Use this SMTP port.  Most users will be fine with the default (25)
 PORT 25
 
 # Path to your alias file.  Just stay with the default.
-ALIASES /etc/mail/aliases
+ALIASES /etc/aliases
 
 # Path to your spooldir.  Just stay with the default.
 SPOOLDIR /var/spool/dma
@@ -39,7 +41,7 @@
 
 # Uncomment if you want to defer your mails.  This is useful if you are
 # behind a dialup line.  You have to submit your mails manually with dma -q
-#DEFER
+DEFER
 
 # Uncomment if you want the bounce message to include the complete original
 # message, not just the headers.
